Firestore/Example/Firestore.xcodeproj/project.pbxproj

```diff
@@ -1864,6 +1533,10 @@
 				);
 				INFOPLIST_FILE = "Firestore/Firestore-Info.plist";
 				MODULE_NAME = ExampleApp;
+				OTHER_LDFLAGS = (
```

Comment:
This is the relevant change not from `pod deprecate`

Reply:
`pod deprecate` doesn't seem to be a thing. Do you mean `pod deintegrate`?

I'm not sure we're seeing much pain from having the project stored integrated. It does mean that as developers we need to run the same version of cocoapods, but that's pretty easy for us to coordinate within the Firestore team.

Meanwhile as we're migrating to C++ we're constantly changing the project to add C++ tests and remove Objective-C tests. It will be a serious pain to have to `pod deintegrate` to even be able to read the diffs of those changes.

I get it that `pod deintegrate`d projects are cleaner, but practically speaking they're much more cumbersome. I think we should leave this installed.

Reply:
Yes I meant `pod deintegrate`.  And undone now.